index.md

```diff
@@ -31,57 +37,52 @@ layout: default
+<object type="image/svg+xml" data="https://cdn.jsdelivr.net/gh/lerna-stack/lerna-stack.github.io@master/img/lerna_content_list_en.drawio.svg"></object>
+
 
 | Name  | Description |
 | ----------- | ---- |
-| [akka-entity-replication](https://github.com/lerna-stack/akka-entity-replication) | Akka extension for fast recovery from failure with replicating stateful entity on multiple nodes in Cluster. |
-| [lerna-http](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-http.md) | Lerna HTTP library provides HTTP related features like below.<br> - Custom Json Format for [spray-json](https://github.com/spray/spray-json) <br> - Custom Directives for [Akka HTTP](https://doc.akka.io/docs/akka-http/current/index.html) <br> - Custom Header for [Akka HTTP](https://doc.akka.io/docs/akka-http/current/index.html) |
-| [lerna-log](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-log.md) | Lerna Log library provides logging related features like below. <br> - Custom log converters for [Logback](http://logback.qos.ch/) |
-| [lerna-management](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-management.md) | Lerna Management library provides management related features like below. <br> - SNMP metrics provider for [Kamon](https://kamon.io/) |
-| [lerna-testkit](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-testkit.md) | Lerna TestKit library provides testkits for <br> - [Akka Classic](https://doc.akka.io/docs/akka/current/index-classic.html) <br> - [Airframe](https://wvlet.org/airframe/) <br> - [WireMock](http://wiremock.org/) |
-| [lerna-util](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util.md) | Lerna Util library provides some utilities like below. <br> - Encryption <br> - Typed Equals <br> - Security <br> - Date and Time |
-| [lerna-util-akka](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util-akka.md) | Lerna Util Akka library provides some utilities related Akka Classic. |
-| [lerna-util-sequence](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-util-sequence.md) | Lerna Util Sequence library provides an ID generator. | 
-| [lerna-validation](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-validation.md) | Lerna Validation library provides custom validators for [Accord](http://wix.github.io/accord/). |
-| [lerna-wart-core](https://github.com/lerna-stack/lerna-app-library/blob/main/doc/lerna-wart-core.md) | Lerna Wart Core library provides custom warts for [WartRemover](https://www.wartremover.org/) |
-| [nablarch-fw-batch-parallelizable](https://github.com/lerna-stack/nablarch-fw-batch-parallelizable) | a library for parallelizing batch processing with [Nablarch Batch Framework](https://nablarch.github.io/docs/LATEST/doc/application_framework/application_framework/batch/index.html) |
+| [application template](https://github.com/lerna-stack/lerna.g8) | a template for creating a new project |
```

Comment:
[IMO] It would be easier to understand if the notation in the contents map image and the contents list table matched.
For example, "Project template" in the image and  "application template" in the table.

Reply:
[memo] new image:

![](https://cdn.jsdelivr.net/gh/lerna-stack/lerna-stack.github.io@feature/add_benchmark/img/lerna_content_list_en.drawio.svg)